src/Sarif.Converters/PylintObjectModel/PylintLogReader.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.IO;
+using System.Text;
+using Newtonsoft.Json;
+
+namespace Microsoft.CodeAnalysis.Sarif.Converters.PylintObjectModel
+{
```

Comment:
I will refactor this logic in a new PR to inherit from abstract class. Using abstract class will need changes in TSLint as well.